iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -321,6 +323,18 @@ public void configureJobConf(TableDesc tableDesc, JobConf jobConf) {
       jobConf.set(opKey, tableDesc.getProperties().getProperty(opKey));
       Preconditions.checkArgument(!tableName.contains(TABLE_NAME_SEPARATOR),
           "Can not handle table " + tableName + ". Its name contains '" + TABLE_NAME_SEPARATOR + "'");
+      if (tableDesc.getTableDescProperty(
+          HiveCustomStorageHandlerUtils.WRITE_OPERATION_CONFIG_PREFIX + tableName) != null) {
+        HiveCustomStorageHandlerUtils.setWriteOperation(jobConf, tableName,
+            Context.Operation.valueOf(tableDesc.getProperties().getProperty(
+                HiveCustomStorageHandlerUtils.WRITE_OPERATION_CONFIG_PREFIX + tableName)));
+      }
+      if (tableDesc.getTableDescProperty(
```

Comment:
please extract config value to local var and reuse

Reply:
Done.

Reply:
is that pushed? why don't you use 
````
bool isMergeTaskEnabled = HiveCustomStorageHandlerUtils.isMergeTaskEnabled(tableDesc.getProperties()::getProperty, String tableName)
````